pkg/composer/composer.go

```diff
@@ -50,7 +51,7 @@ type Options struct {
 	DebugPrintFull bool // full debug print, may leak secret env var to logs
 }
 
-func New(o Options) (*Composer, error) {
+func New(o Options, client *containerd.Client) (*Composer, error) {
```

Comment:
Can we put `client` to `Options` struct?

Reply:
I prefer to separate them, they don’t seem to be affiliation.